tools/azure-devtools/src/azure_devtools/perfstress_tests/random_stream.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+def get_random_bytes(buffer_length):
```

Comment:
Since this is a one-line function call either way, I would be fine directly using `os.urandom()` in the test code.  Or we can keep this helper function.  Whatever you prefer.

Reply:
I might keep the function for now in case I want to later use `bytearray` instead of `bytes`